docs/sql-ref-syntax-qry-select-transform.md

```diff
@@ -24,6 +24,14 @@ license: |
 The `TRANSFORM` clause is used to specify a Hive-style transform query specification 
 to transform the inputs by running a user-specified command or script.
 
+Currently, Spark's script transform support two modes:
+
+    1. Without Hive: Spark scripts transform can run without `-Phive` or `SparkSession.builder.enableHiveSupport()`.
```

Comment:
You wan't have to say the profiles here since end users don't use or know the profiles. You can just say Hive support enabled vs Hive support disabled. And then, you can separately mention both `SparkSession.builder.enableHiveSupport` and `spark.sql.catalogImplementation` configuration

Reply:
How about current?